src/domain/entities/AnalyticsConfig.ts

```diff
@@ -0,0 +1,3 @@
+import { Maybe } from "../../types/utils";
+
+export type AnalyticsConfig = { codeUrl: Maybe<string> };
```

Comment:
I assume that the creation of a separate `AnalyticsConfig` outside the `Config` entity is the effect of actually not having the representation of the "constants" we have as domain settings, by the misuse of retrieving and setting them on the `ConfigRepository` (which indeed is not the correct way of doing it). But at the same time, we're declarating Matomo Analytics and Google Analytics as separate things, something that in my opinion, I would merge. So we could delete the misuse of getting and setting `googleAnalyticsCode` on `ConfigRepository` also, and add `gaCode` here on `AnalyticsConfig`. What do you think?